### PR TITLE
Taskboard 

### PR DESCRIPTION
**Pull request by TruaShamu**
From discord:
**Gamma** https://usaco.guide/bronze/ad-hoc?lang=java Remove the LeetCode problem.
**Delta** https://usaco.guide/silver/prefix-sums?lang=java Get rid of the problems from Leetcode and GoogleKS.
**Epsilon** Get rid of the leetcode problem. https://usaco.guide/silver/prefix-sums-2?lang=java/
**Eta** Get rid of the SAPO problem https://usaco.guide/silver/intro-ordered?lang=java
**Theta**  https://usaco.guide/silver/greedy-sorting?lang=java get rid of the csacademy problem
**Xi** https://usaco.guide/silver/func-graphs?lang=cpp Remove the oj.uz problem

### CODE REVIEW

File: content/3_Silver/Func_Graphs.problems.json [hunk 1]
```diff
@@ -55,17 +55,6 @@
...
-      "url": "https://oj.uz/problem/view/IOI11_garden",
-      "source": "oj.uz",
-      "difficulty": "Hard",
-      "isStarred": false,
-      "tags": ["Functional Graph"],
-      "solutionMetadata": {
-        "kind": "none"
-      }
-    }
+
```
Comment (maggie-j-liu) [inline content/3_Silver/Func_Graphs.problems.json:58]: i think these extra lines might be breaking the build? idk

**Comment by TruaShamu** [comment]
Alright, I removed it.

File: content/3_Silver/Func_Graphs.problems.json (hunk 1 above)
Comment (maggie-j-liu) [reply content/3_Silver/Func_Graphs.problems.json]: in the other files too

**Comment by maggie-j-liu** [comment]
can we add the problems with internal solutions to extraProblems.json?

File: content/3_Silver/Intro_Ordered.problems.json [hunk 2]
```diff
@@ -140,17 +140,5 @@
         "site": "CF"
       }
     },
```
Comment (maggie-j-liu) [inline content/3_Silver/Intro_Ordered.problems.json:142]: oops, i think it's actually the comma here

File: content/3_Silver/Func_Graphs.problems.json [hunk 3]
```diff
@@ -55,17 +55,5 @@
         "kind": "internal"
       }
     },
```
Comment (maggie-j-liu) [inline content/3_Silver/Func_Graphs.problems.json:57]: and here

**Review by brad182 — APPROVED** [review]

**Review by maggie-j-liu — APPROVED** [review]

**Review by nwatx — APPROVED** [review]

**Review by dongliu0426 — APPROVED** [review]

**Pull request merged**